backend/tests/services/execution/test_request_builder_task_data.py

```
@@ -0,0 +1,286 @@
+
+from shared.models.execution import ExecutionRequest
+
+# Constants for test data
+TEST_USER_ID = 42
+TEST_USER_NAME = "testuser"
+TEST_TASK_ID = 100
+TEST_SUBTASK_ID = 200
+TEST_MESSAGE_ID = 300
+TEST_TEAM_ID = 10
+TEST_BOT_ID = 5
+TEST_GIT_TOKEN = "ghp_test_token"
```

Ruff (0.15.2)

[error] Possible hardcoded password assigned to: "TEST_GIT_TOKEN"

(S105)